backend/src/main/java/com/google/sps/data/TimeHelper.java

```diff
@@ -1,42 +1,37 @@
 package com.google.sps.data;
 
-import java.util.Date;
-import java.util.Calendar;
-import java.time.LocalDateTime;
+import java.time.ZonedDateTime;
 
-/** Class used to set reference date/time for testing and convert input timeAvailableUntil to date timestamp */
+/**
+ * Class used to set reference date/time for testing and convert input timeAvailableUntil to date
+ * timestamp
+ */
 public final class TimeHelper {
 
-  private static Date date;
+  private static ZonedDateTime dateTime;
 
   /** Constructor */
   public TimeHelper() {
-    this(new Date());
+    this(ZonedDateTime.now());
   }
 
   /** Constructor with manually set date */
-  public TimeHelper(Date date) {
-    this.date = date;
+  public TimeHelper(ZonedDateTime dateTime) {
+    this.dateTime = dateTime;
   }
 
-  /** Return today's date with time of hours:minutes in milliseconds since epoch */
-  public static long getDateMillis(int hours, int minutes) {
-    // Validate hours and minutes
-    if (hours < 0 || hours >= 24) {
+  /** Return today's date with time of hour:minute */
+  public static ZonedDateTime getNewTimeToday(int hour, int minute) {
+    // Validate hour and minute
+    if (hour < 0 || hour >= 24) {
       throw new IllegalArgumentException("Hours can only be 0 through 23 (inclusive).");
     }
-    if (minutes < 0 || minutes >= 60) {
+    if (minute < 0 || minute >= 60) {
       throw new IllegalArgumentException("Minutes can only be 0 through 59 (inclusive).");
     }
```

Comment:
will .withHour() and .withMinute() throw in these circumstances? If so you might be able to just defer to those methods to throw the necessary exceptions.

Reply:
Good point, they throw DateTimeExceptions. I removed the manual checks and changed the TimeHelperTest.

Reply:
Thanks!